### PR TITLE
Timeout

### PR DESCRIPTION
**Pull request by ashleypittman**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -117,8 +117,8 @@ pipeline {
                                              aggregatingResults: true,
                                              id: "analysis-centos7",
                                              tools: [ gcc4(), cppCheck() ],
-                                             filters: [excludeFile('.*\\/_build\\.external-Linux\\/.*'),
-                                                       excludeFile('_build\\.external-Linux\\/.*')]
+                                             filters: [excludeFile(".*\\/_build\\.external${arch}\\/.*"),
+                                                       excludeFile("_build\\.external${arch}\\/.*")]
```
Comment (brianjmurrell) [inline Jenkinsfile]: Unless the [RFE that I filed upstream](https://issues.jenkins-ci.org/browse/JENKINS-55574) has been completed (but I didn't see my ticket closed) substituting `${arch}` doesn't work here.

File: Jenkinsfile [hunk 2]
```diff
@@ -145,8 +145,8 @@ pipeline {
                                              aggregatingResults: true,
                                              id: "analysis-ubuntu18",
                                              tools: [ gcc4(), cppCheck() ],
-                                             filters: [excludeFile('.*\\/_build\\.external-Linux\\/.*'),
-                                                       excludeFile('_build\\.external-Linux\\/.*')]
+                                             filters: [excludeFile(".*\\/_build\\.external${arch}\\/.*"),
+                                                       excludeFile("_build\\.external${arch}\\/.*")]
```
Comment (brianjmurrell) [inline Jenkinsfile]: Ditto.

File: Jenkinsfile [hunk 3]
```diff
@@ -192,12 +195,20 @@ pipeline {
                             junit 'nosetests-centos.xml'
                             archiveArtifacts artifacts: '**/*.log'
                         }
+                        cleanup {
+                        dir('test/output') {
```
Comment (brianjmurrell) [inline Jenkinsfile]: Is GitHub just not displaying the indentation properly here?

File: Jenkinsfile (hunk 1 above)
Comment (ashleypittman) [reply Jenkinsfile]: I think this is the difference between ' and ", it's simply how strings work in groovy.

File: Jenkinsfile (hunk 3 above)
Comment (ashleypittman) [reply Jenkinsfile]: Probably tabs vs spaces, I did try to correct it but this file is all over the place anyway.

**Pull request merged**